.github/workflows/e2e-test-linux-vm.yaml

```diff
@@ -439,7 +439,9 @@ jobs:
           --set githubConfigUrl="https://github.com/${{ steps.resolved_inputs.outputs.TARGET_ORG }}/${{steps.resolved_inputs.outputs.TARGET_REPO}}" \
           --set githubConfigSecret.github_token="${{ steps.setup.outputs.token }}" \
           --set containerMode.type="kubernetes" \
+          --set containerMode.kubernetesModeWorkVolumeClaim.accessModes={"ReadWriteOnce"} \
```

Comment:
some defaults value get removed from https://github.com/actions/actions-runner-controller/pull/2416/files

Reply:
That is why k8s mode failed without being modified here... Got it!